src/main/scala/daisy/opt/NNQuantizationPhase.scala

```diff
@@ -239,7 +239,7 @@ object NNQuantizationPhase extends DaisyPhase with tools.RoundoffEvaluators with
 
     val f: Future[Unit] = Future {
       // Run soplex with the model until timeout
-      val res = Runtime.getRuntime().exec("/var/tmp/qnn/scipoptsuite-7.0.3/scip_build/bin/scip -f model.lp -s scip.set")
+      val res = Runtime.getRuntime().exec("/var/tmp/safe-qnn/scipoptsuite-7.0.3/scip_build/bin/scip -f model.lp -s scip.set")
```

Comment:
This also will not be changed. Revert it back to the original. I changed it to match the path for the machine we had.